omero/omero-monitoring-agents.yml

```diff
@@ -60,23 +60,45 @@
       content: "{{ secret_monitoring_nginx_htpasswd | default(monitoring_nginx_htpasswd) }}"
       dest: /etc/nginx/monitoring.htpasswd
 
+  vars:
+    # monitoring:monitoring
+    monitoring_nginx_htpasswd: |
+      monitoring:$apr1$njrafrtU$19wf/I15zPuSudlM5Y50Z0
```

Comment:
Safe enough for this to be in public view?

Reply:
Yes, this is a public example. It's overridden by a private var